simpleocr/files.py

```diff
@@ -45,45 +32,152 @@ def write(self):
         write_boxfile(self.path, self.classes, self.segments)
 
 
-class ImageFile(object):
```

Comment:
Well, I did consider doing an `Image` class with a `Ground` class and then the `ImageFile` and `GroundFile` child-classes, but I didn't choose this option because I thought it would be less clear to the user in the end. Still, I'd agree that a single class that can or can not have a file attached to it isn't really elegant either, so I don't really mind either way.